core/src/com/unciv/logic/civilization/PolicyManager.kt

```diff
@@ -140,12 +141,19 @@ class PolicyManager {
     }
 
     fun tryAddLegalismBuildings() {
+        val maxBuildings = 4        // single place should we wish to make rule details part of the loadable ruleset
+        if (legalismState.size >= maxBuildings) return
+        // Two filters: Cities failing the tests in the first allow a later city to be considered instead
+        // Failure of the test(s) in the second cause the city to 'reserve' a slot but delay the benefit until those conditions are met.
         val candidateCities = civInfo.cities
-                .sortedBy { it.turnAcquired }
-                .subList(0, min(4, civInfo.cities.size))
                 .filter { it.id !in legalismState
-                        && it.cityConstructions.hasBuildableCultureBuilding() }
-        for (city in candidateCities) {
+                            && !it.isPuppet
+                            && !it.isInResistance() }
+        val benefitCities = candidateCities
+                .sortedBy { it.turnAcquired }
+                .subList(0, min(maxBuildings - legalismState.size, candidateCities.size))
+                .filter { it.cityConstructions.hasBuildableCultureBuilding() }
```

Comment:
This comment suggests that conquered cities do not count for legalism policy, so i think we should just filter out non founded cities: https://www.reddit.com/r/civ/comments/2ju1c2/what_does_first_four_cities_mean/clfbc24/

if we agree the cities must be founded, is there a point in this change?

Reply:
reddit is broken for me, the 'view entire...' button does nothing (yes, I whitelisted reddit in ublock). But if what the visible text says is right, then neither what you say not any version I coded is correct.